chart-values/kibana_values.yaml

```diff
@@ -66,15 +66,16 @@ service:
 ingress:
   enabled: true
   hosts:
-    - kibana-central.mattermost.com
-    # - localhost.localdomain/kibana
+    - kibana.example.com
   annotations:
     kubernetes.io/ingress.class: nginx
     certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/auth-url: "https://auth.example.com/oauth2/auth"
+    nginx.ingress.kubernetes.io/auth-signin: "https://auth.example.com/oauth2/start?rd=https://$host$request_uri"
   tls:
     - secretName: kibana-server-tls
       hosts:
-        - kibana-central.mattermost.com
+        - kibana.example.com
```

Comment:
this is correct?

Reply:
@cpanato Yeap, I thought its better to keep some example values in these chart values as the central monitoring ones are being stored in another repo.